_get_keyboard.py

```diff
@@ -0,0 +1,9 @@
+# 此脚本返回 0 - 9 的所有数值, 似乎无法正确判断keyboard
```

Comment:
comments in English are more appreciated, it's not my native language neither but it's widely understood

Reply:
You are right